package.json

```
@@ -133,6 +133,16 @@
             false
           ]
         },
+        "deno.cacheOnSave": {
+          "type": "boolean",
+          "default": true,
```

```suggestion
          "default": false,
```

Let's make this false to start and get feedback on.